src/components/Editor/Editor.scss

```diff
@@ -39,34 +39,35 @@ limitations under the License.
     align-items: center;
     justify-content: center;
     text-align: center;
-    color: $dark-blue;
+    color: $kubeapps-text;
     font-size: 1.2em;
   }
   .editorHeader {
     text-align: left;
-    padding: 0 1em;
-    border-bottom: 1px solid $light-gray;
+    border-bottom: 1px solid $kubeapps-light;
     .title {
       display: inline-block;
-      font-size: 1.2em;
-      opacity: 0.5;
     }
     .editingFlag {
       width: 10px;
       height: 10px;
       border-radius: 5px;
-      background: $blue;
+      background: $kubeapps-accent;
       display: inline-block;
       margin-left: 5px;
     }
   }
   .editorFooter {
     display: flex;
-    border-top: 1px solid $light-gray;
+    border-top: 1px solid $kubeapps-light;
     flex-direction: column;
     z-index: 10;
     a {
-      color: $dark-blue;
+      color: $kubeapps-text;
+    }
+    .fa {
+      color: $kubeapps-text !important;
+      font-size: 18px !important;
```

Comment:
I'd rather not mix `px` and `em` unit in font-sizes. Could you use `em` or `rem` at least?